libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.AdaptiveSkillDialog.schema

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.IDialog)",
```

Comment:
>union [](start = 14, length = 5)

this may have changed to "implements(Microsoft.IDialog)" check when you merge with master.

Reply:
Done